src/App.css

```diff
@@ -30,4 +30,7 @@ input[type="text"] {
   grid-template-columns: 1fr 1fr 1fr;
   grid-template-rows: 1fr 1fr 1fr;
   gap: 15px 15px;
+  padding: 0;
 }
+
+.foodListItems {margin:auto}
```

Comment:
no, add it here: https://github.com/code-knackerin/ShoppingList/blob/main/src/components/List.css
`foodListItems` is in a different component.

Reply:
okay

Reply:
> no, add it here: https://github.com/code-knackerin/ShoppingList/blob/main/src/components/List.css `foodListItems` is in a different component.

Done